tchannel/serializer/json.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import json
+
+from tchannel.schemes import JSON
+
+
+class JsonSerializer(object):
+    def __init__(self):
+        self.name = JSON
+
+    def deserialize_header(self, obj):
+        if not obj:
+            return None
+        return json.loads(obj)
+
+    def serialize_header(self, obj):
+        if not obj:
+            obj = {}
```

Comment:
Did we decide that `{}` was the default value for headers and body? I don't get why `tchannel.json(body)` would also send `"{}"` even though I haven't set anything there.